File: aat-lib/src/main/java/ch/bailu/aat_lib/gpx/GpxInformation.java
package ch.bailu.aat_lib.gpx;

import static ch.bailu.foc.Foc.FOC_NULL;

import ch.bailu.foc.Foc;

public class GpxInformation extends GpxDataWrapper  {
    public final static GpxInformation NULL=new GpxInformation();

    public boolean isLoaded() {
        return false;
    }
    public GpxList getGpxList() {
        return GpxList.NULL_TRACK;
    }
    public float getAccuracy() {
        return 0f;
    }
    public int getState() {return StateID.ON;}
    public Foc getFile() {return FOC_NULL; }
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/gpx/GpxInformation.kt
package ch.bailu.aat_lib.gpx

import ch.bailu.foc.Foc

open class GpxInformation : GpxDataWrapper() {
    open fun getLoaded(): Boolean {
        return false
    }

    open fun getGpxList(): GpxList {
        return GpxList.NULL_TRACK
    }

    open fun getAccuracy(): Float {
        return 0f
    }

    open fun getState(): Int {
        return StateID.ON
    }

    open fun getFile(): Foc {
        return Foc.FOC_NULL
    }

    companion object {
        @JvmField
        val NULL: GpxInformation = GpxInformation()
    }
}
